src/switchcraft_advanced/services/intune_service.py

```
@@ -0,0 +1,362 @@
+        app_info: dict with keys: displayName, description, publisher, installCommandLine, uninstallCommandLine
+        """
+        import zipfile
+        import xml.etree.ElementTree as ET
+        import base64
+
+        headers = {"Authorization": f"Bearer {token}", "Content-Type": "application/json"}
+        base_url = "https://graph.microsoft.com/beta/deviceAppManagement" # Use beta for win32LobApp usually
+
+        intunewin_path = Path(intunewin_path)
+        if not intunewin_path.exists():
+            raise FileNotFoundError(f"File not found: {intunewin_path}")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)